pages/hardware/i2c-and-spi.md

```diff
@@ -101,7 +111,7 @@ __Note:__ The 90 and 270 degrees rotation options require additional memory on G
 so won't work with the 16M GPU split.
 
 ### Customising config.txt
-These are some tips and tricks for customizing your raspberry pi. Most of them require changing settings the the config.txt file on the SD cards `boot` partition.
+These are some tips and tricks for customizing your raspberry pi. Most of them require changing settings the the config.txt file on the SD cards `boot` partition. See [here](https://docs.resin.io/configuration/advanced/) for more details
```

Comment:
Use relative links only as the domain name might change: `/configuration/advanced/`